Sources/Commands/SwiftTestTool.swift

```diff
@@ -195,7 +195,7 @@ public struct SwiftTestTool: SwiftCommand {
 
             // validate XCTest available on darwin based systems
             let toolchain = try swiftTool.getTargetToolchain()
-            if toolchain.targetTriple.isDarwin() && toolchain.xctestPath == nil {
+            if try (toolchain.targetTriple.isDarwin() && toolchain.xctestPath == nil) || !swiftTool.getHostToolchain().swiftSDK.supportsTesting {
```

Comment:
readability and line length nit
```suggestion
            let isHostTestingAvailable = try swiftTool.getHostToolchain().swiftSDK.supportsTesting
            if (toolchain.targetTriple.isDarwin() && toolchain.xctestPath == nil) || !isHostTestingAvailable {
```